phylogenetic/config/exclude.txt

```diff
@@ -59,3 +59,838 @@ AY656168 # circular SYN 14-JAN-2005
+U88536 # record withdrawn from NCBI
+JC562941 # PAT and duplicate of NC_001477
+LP918031 # PAT and duplicate of NC_001477
+LQ729831 # PAT and duplicate of NC_001477
+LQ868375 # PAT and duplicate of NC_001477
+LQ868392 # PAT and duplicate of NC_001477
+LY921396 # PAT and duplicate of NC_001477
+OG164702 # PAT and duplicate of NC_001477
+OG164733 # PAT and duplicate of NC_001477
+MW828678 # duplicate of NC_001477
+MZ284953 # duplicate of NC_001477
+MZ285732 # duplicate of NC_001477
+MP552823 # PAT and duplicate of KX812530
+LQ905241 # PAT
+LX250395 # PAT and duplicate of LQ905241
+LY590179 # PAT and duplicate of LQ905241
+HZ784932 # PAT
+LY511421 # PAT and duplicate of HZ784932
+MA780861 # PAT and duplicate of HZ784932
+LP918033 # PAT and duplicate of AY744147
+LZ239282 # PAT and duplicate of AB074760
+FV537260 # PAT
+FV537261 # PAT and duplicate of FV537260
+MP552819 # PAT and duplicate of KU725663
+FV537268 # PAT
+FV537269 # PAT and duplicate of FV537268
+OG158724 # PAT and duplicate of KC333651
+U88535 # record withdrawn from NCBI
+HH961660 # PAT
+JE963510 # PAT and duplicate of HH961660
+FV537258 # PAT
+FV537259 # PAT and duplicate of FV537258
+JE963511 # PAT and duplicate of AY037116
+CS477264 # PAT and duplicate of MT076932
+DI401606 # PAT and duplicate of MT076932
+DM140253 # PAT and duplicate of MT076932
+MP552817 # PAT and duplicate of MT076932
+AF180817 # duplicate of MT076932
+MT076932 # duplicate of MT076932
+MT076933 # duplicate of MT076932
+MT076934 # duplicate of MT076932
+MT076935 # duplicate of MT076932
+LP918029 # PAT and duplicate of AF326573
+LG230782 # PAT
+LQ243187 # PAT and duplicate of LG230782
+LZ951197 # PAT and duplicate of LG230782
+MP451520 # PAT and duplicate of LG230782
+FV537262 # PAT
+FV537263 # PAT and duplicate of FV537262
+CS477305 # PAT
+CS479205 # PAT and duplicate of CS477305
+CS805345 # PAT and duplicate of CS477305
+DI401646 # PAT and duplicate of CS477305
+DI403035 # PAT and duplicate of CS477305
+DM140293 # PAT and duplicate of CS477305
+DM140334 # PAT and duplicate of CS477305
+FB667400 # PAT and duplicate of CS477305
+FV537254 # PAT
+FV537255 # PAT and duplicate of FV537254
+FV537256 # PAT
+FV537257 # PAT and duplicate of FV537256
+MP552821 # PAT and duplicate of KU725665
+LY683284 # PAT and duplicate of KF887994
+LQ905237 # PAT
+LX250391 # PAT and duplicate of LQ905237
+LY590175 # PAT and duplicate of LQ905237
+LY683286 # PAT and duplicate of KP406805
+LQ905238 # PAT
+LX250392 # PAT and duplicate of LQ905238
+LY590176 # PAT and duplicate of LQ905238
+LQ905242 # PAT
+LX250396 # PAT and duplicate of LQ905242
+LY590180 # PAT and duplicate of LQ905242
+HH961661 # PAT and duplicate of AF038403
+CS805348 # PAT
+FB667404 # PAT and duplicate of CS805348
+FV537264 # PAT
+FV537265 # PAT and duplicate of FV537264
+FV537266 # PAT
+FV537267 # PAT and duplicate of FV537266
+OG158723 # PAT and duplicate of GQ466079
+M93130 # record withdrawn from NCBI
+LP918034 # PAT
+M19197 # record withdrawn from NCBI
```

Comment:
Double check some of the withdrawn records. For example, this [M19197](https://www.ncbi.nlm.nih.gov/nuccore/M19197) does not match the LOCUS field `DENRCG`

Reply:
Done with https://github.com/nextstrain/dengue/pull/29/commits/6932f11c199ca38f010c2ce60e264cdf4f714ecb